documentation/source/users/arkane/input.rst

```diff
@@ -688,12 +831,135 @@ with the reaction label, delineates the semi-normalized sensitivity coefficients
+
+Pressure Dependent Rate Calculation
+===================================
+
+The overall parameters for the pressure-dependence calculation must be defined in a
+``pressureDependence()`` function at the end of the input file. The following parameters are necessary:
+
+================================================== ====================================================================================================================================================
+Parameter                                          Description
+================================================== ====================================================================================================================================================
+``label``                                          Use the name for the ``network`` declared previously
+``method``                                         Method to use for calculating the pdep network. Use either ``'modified strong collision'``, ``'reservoir state'``, or ``'chemically-significant eigenvalues'``
+``interpolationModel``                             Select the output type for the pdep kinetics, either in ``'chebyshev'`` or ``'pdeparrhenius'`` (plog) format
+``activeKRotor``                                   A flag indicating whether to treat the K-rotor as active or adiabatic
+``activeJRotor``                                   A flag indicating whether to treat the J-rotor as active or adiabatic
+``Tmin``/``Tmax``/``Tcount`` **or** ``Tlist``      Define temperatures at which to compute (and output) :math:`k(T,P)`
+``Pmin``/``Pmax``/``Pcount`` **or** ``Plist``      Define pressures at which to compute (and output) :math:`k(T,P)`
+``maximumGrainSize`` **and** ``minimumGrainCount`` Defines fineness of energy grains used in master equation calculations.
+``sensitivity_conditions``                         Specifies the conditions at which to run a network sensitivity analysis.
+================================================== ====================================================================================================================================================
+
+Temperature and Pressure Ranges
+-------------------------------
+
+Arkane will compute the :math:`k(T,P)` values on a grid of temperature and
+pressure points. ``Tmin``, ``Tmax``, and ``Tcount`` values, as well as ``Pmin``, ``Pmax``, and ``Pcount`` parameter
+values must be provided. Arkane will automatically choose the intermediate temperatures based on the interpolation model
+you wish to fit. This is the recommended approach.
+
+Alternatively, the grid of temperature and pressure points can be specified explicitly using ``Tlist`` and/or ``Plist``.
+
+
+Energy Grains
+-------------
+
+Determine the fineness of the energy grains to be used in the master equation calculations.  Dictate
+the ``maximumGrainSize``, and the ``minimumGrainCount``.
+
+
+An example of the algorithm parameters function for the acetyl + O2 network is shown below.
```

Comment:
Maybe add a `Pressure-dependent sensitivity analysis` title before the example?

Reply:
I added more description of sensitivity analysis and made it its own section. Since the example seems to cover all the syntax for this method (as opposed to just sensitivity analysis syntax), I moved the example closer to the description of all the parameters and referenced it in the text of the sensitivity analysis section. See the second commit in this PR.